src/renderer/src/routes/modals/NetworkNodeSelection/index.tsx

```diff
@@ -129,6 +130,13 @@ const NetworkNodeSelection = () => {
                   </div>
 
                   <div className="flex-start flex min-w-0 grow flex-col gap-0.5">
+                    {blockchain === 'neox' &&
+                      BSNeoXConstants.ANTI_MEV_RPC_LIST_BY_NETWORK_ID[network.id].some(url => url === node.url) && (
+                        <span className="bg-neon/70 text-asphalt text-1xs mb-0.5 block w-fit rounded px-1.25 py-px text-center font-semibold">
+                          {t('antiMevLabel')}
+                        </span>
+                      )}
```

Comment:
What do you think about creating a item component so you can create a variable like `isNeoXAntiMev` ?